dev-docs/DEV_ENVIRONMENT.md

```diff
@@ -1,4 +1,4 @@
-# A2 Development Environment
+# Automate Development Environment
```

Comment:
I changed most of the references to A2 in this document to Automate to reflect the current state of the project. There are many references to `a2-dev.test` that I did not change because that is likely a larger effort.